### ISSUE TITLE
continuous integration and testing

### ISSUE THREAD

**Issue by aaronspring** [pankajkarman/bias_correction#10]
great package. I am wrapping it in https://github.com/pangeo-data/climpred/blob/355f849d19838085705c89d7b912b0e2fae89f07/climpred/bias_removal.py#L492 and it will help us bias correcting precip forecasts.

Is there a way to test `bias_correction` (in a continuous integration sense)?

- what/how: Making some demo fake data with different distributions and then running `bias_correction` on it and see how whether the results are as expected
- why: stability of the package, implement new features without breaking the rest, ...

**Comment by aaronspring** [comment]
@PauRiv

**Comment by pankajkarman** [comment]
I have been thinking about implementing some test cases to ensure the robustness of the package. However, I am not sure about how to do it. May be take data from papers already using or proposing the methods implemented here.

**Comment by aaronspring** [comment]
couldnt we create a normal distribution for observations, add some skew or mean shift to get observations and then test that after bias reduction the method='normal_mapping/basic_quantile/modified_quantile' bias-reduced forecasts are on a straight line with observations? 

no clue what to do with `gamma_mapping`

**Comment by pankajkarman** [comment]
yeah. That's good one. We have to think over the ways to test gamma mapping.
